### ISSUE TITLE
Feature suggestion - sleep timer

### ISSUE THREAD

**Issue by wturrell** [coderholic/pyradio#163]
I'm surprised no-one seems to have suggested this yet…

You need a way to set the timer (perhaps give people a choice of relative time - e.g. 15 minutes - and absolute - 2am) and then the time playback will stop is shown in the status bar.

Thanks!

**Comment by s-n-g** [comment]
Hm...
This is a nice one too!
Let me repeat myself: "let me think about it" :)

**Comment by amano-kenji** [comment]
Isn't it a good idea to add shutdown remote server command? Scheduling can be done externally.

**Comment by amano-kenji** [comment]
A few terminal options that can send remote commands

* `sleep 5; send-remote-command`
* `termdown 5m; send-remote-command`

This is dirty, but it does the job.

**Comment by s-n-g** [comment]
Yes, there are many ways to achieve this feature, but I've been trying to make it to integrate it into the program

It keeps gets postponed as other things intervene, but we'll see how it goes.

**Comment by amano-kenji** [comment]
This should be a scheduler command.
